recipes/rotor/all/conanfile.py

```diff
@@ -49,7 +50,7 @@ def configure(self):
                 pass
 
     def requirements(self):
-        self.requires("boost/1.81.0")
+        self.requires("boost/1.83.0", transitive_headers=True)
```

Comment:
Is Boost exposed now in a public header? If yes, could you please point where? We try to avoid transitive dependencies, unless they are really needed.

Reply:
Yes, it publicly depends on boost headers

https://github.com/basiliscos/cpp-rotor/blob/master/include/rotor/arc.hpp#L27
https://github.com/basiliscos/cpp-rotor/blob/master/include/rotor/message.h#L115